pkg/hintrunner/zero/zerohint_memcpy.go

```diff
@@ -3,9 +3,54 @@ package zero
 import (
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/core"
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+
 	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
 )
 
+func newMemcpyContinueCopyingHint(output hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "MemcpyContinueCopying",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> n -= 1
+			//> ids.continue_copying = 1 if n > 0 else 0
+
+			//> n-=1
+			n, err := ctx.ScopeManager.GetVariableValue("n")
+			if err != nil {
+				return err
+			}
+
+			ctx.ScopeManager.AssignVariable("n", n.(uint64)-1)
+
+			//> ids.continue_copying = 1 if n > 0 else 0
+			continueCopyingAddr, err := output.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			var v memory.MemoryValue
+			if n.(uint64) > 0 {
+				v = memory.MemoryValueFromFieldElement(&utils.FeltZero)
```

Comment:
Inversion of FeltZero and FeltOne
I also inverted in my comment sorry, i modified it now.

hint is : 
```
//> ids.continue_copying = 1 if n > 0 else 0
```

so : 
```
if n.(uint64) > 0 {
	v = memory.MemoryValueFromFieldElement(&utils.Feltone)
} ...
```

Reply:
Solved! ✅